routes/admin.php

```diff
@@ -89,7 +89,7 @@
         Route::module('digitalPublications.articles');
 
         // WEB-1963: Browser for nested modules must be implemented manually
-        Route::get('digitalPublicationsFoo/{digitalPublication}/articles/browser', [DigitalPublicationArticleController::class, 'browser'])->name('collection.articles_publications.digitalPublications.articles.subbrowser');
+        Route::get('digitalPublicationsFoo/{digitalPublication}/articles/browser', [DigitalPublicationArticleController::class, 'getSubbrowserItems'])->name('collection.articles_publications.digitalPublications.articles.subbrowser');
```

Comment:
Let's remove `Foo` from the route here.

Reply:
Foo is here to prevent the twill middleware from routing to the edit page. Nested modules don't play well with url resolution so removing Foo from this routes to /browser/edit?digitalPublication={id} instead of /browser because it has a matching model it thinks it can resolve down.

Reply:
Ah, got it. Looks good then!